<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiffToolDialog</class>
 <widget class="QDialog" name="DiffToolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diff Tool</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin" stdset="0">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin" stdset="0">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Layout A   </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Layout B</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="lay::CellViewSelectionComboBox" name="layouta">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="lay::CellViewSelectionComboBox" name="layoutb">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin" stdset="0">
       <number>9</number>
      </property>
      <item>
       <widget class="QCheckBox" name="smart_cbx">
        <property name="text">
         <string>Don't use names to match cells (use geometrical properties)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="xor_cbx">
        <property name="text">
         <string>Run XOR on differences</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="summarize_cbx">
        <property name="text">
         <string>Summarize missing layers</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="detailed_cbx">
        <property name="text">
         <string>Detailed information</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="expand_cell_arrays_cbx">
        <property name="text">
         <string>Expand cell arrays (compare single instance by instance)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="exact_cbx">
        <property name="text">
         <string>Exact compare (includes properties, text orientation and similar)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ignore_duplicates_cbx">
        <property name="text">
         <string>Ignore duplicate instances and shapes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>472</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lay::CellViewSelectionComboBox</class>
   <extends>QComboBox</extends>
   <header>layWidgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiffToolDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiffToolDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
